packages/salesforcedx-vscode-core/src/commands/commands.ts

```diff
@@ -316,14 +317,11 @@ export interface CommandletExecutor<T> {
 
 export abstract class SfdxCommandletExecutor<T>
   implements CommandletExecutor<T> {
-  public execute(response: ContinueResponse<T>): void {
-    const cancellationTokenSource = new vscode.CancellationTokenSource();
-    const cancellationToken = cancellationTokenSource.token;
-
-    const execution = new CliCommandExecutor(this.build(response.data), {
-      cwd: vscode.workspace.rootPath
-    }).execute(cancellationToken);
-
+  protected attachExecution(
```

Comment:
I think this needs a newline before here. I'll be honest and say that I don't like to leave such comments in code reviews. I'm open to spending time using a better code formatter. But as far as I know, many code formatters cannot insert new lines between fields, methods.